### ISSUE TITLE
False positive for LiteralAsCondition with backticks

### ISSUE THREAD

**Issue by mvz** [bbatsov/rubocop#4905]
The LiteralAsConditon cop also considers a string in backticks a literal. This is wrong, as its 'value' is the result of running the contents as a command.

For example, I have this in my code:

```ruby
      unless `intltool-merge --version`
        raise Exception, 'Need to install intltool'
      end
 ```

## Expected behavior

RuboCop does not complain.

## Actual behavior

RuboCop complains with:

```
util/rake/gettextgenerate.rb:72:14: W: Literal intltool-merge --version appeared as a condition.
      unless `intltool-merge --version`
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

Create a file with the above contents, run rubocop on it.

## RuboCop version

```
0.51.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x86_64-linux)
```

**Comment by pocke** [comment]
I think RuboCop should complain about it.
Because backtick literal always returns a string or raises an error.

```ruby
p `echo foo` # => "foo\n"

begin
  `unknwon-command`
rescue => ex
  p ex # => #<Errno::ENOENT: No such file or directory - unknwon-command>
end
```

**Comment by mvz** [comment]
Ah yes, the code is wrong. Good catch :-).
